programmers/난이도별/level01.K번째수/rockmiin.py

```diff
@@ -0,0 +1,9 @@
+
+def solution(array, commands):
+
+    result=[]
+    for i in range (len(commands)):
+        cut=array[commands[i][0]-1:commands[i][1]]
+        cut=sorted(cut)
+        result.append(cut[commands[i][2]-1])
+    return result
```

Comment:
그리고 이 문제는 한 줄로 표현 가능합니다.
```suggestion
    return [ sorted(arrays[i - 1:j])[k - 1] for i, j, k in commands ];
```

Reply:
이렇게 표현할 줄은 몰랐네요 감사합니다 :)